### DISCUSSION TITLE
Problems with using the external ttkbootstrap module

### DISCUSSION THREAD

**Discussion by lucas-ita** [cwapi3d/cwapi3dpython#165] [Q&A]
When I use the `ttkbootstrap `library, some problems arise. The axis in Cadwork changes color and I can't select objects, and the selection is irregular.

Axis:
![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/87e9c403-848c-49c1-81a2-c7663a5f821a)

Selection:
![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/0f93b811-7c58-4165-9c5b-e8ee2c24c85c)

**Comment by Brunner246** [comment]
don't you need a `mainloop`?

Have u tried their example (see below) ?

```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *

root = ttk.Window(themename="superhero")

b1 = ttk.Button(root, text="Submit", bootstyle="success")
b1.pack(side=LEFT, padx=5, pady=10)

b2 = ttk.Button(root, text="Submit", bootstyle="info-outline")
b2.pack(side=LEFT, padx=5, pady=10)

root.mainloop()
```
